packages/module/src/ConsoleShared/src/components/spotlight/spotlight.scss

```diff
@@ -56,7 +56,7 @@
       bottom: 0;
       animation: 1.2s pfext-spotlight-expand 1.6s ease-out;
       animation-fill-mode: forwards;
-      outline: 4px solid var(--pf-v5-global--palette--blue-200);
+      outline: 4px solid var(--pf-v6-global--palette--blue-200);
```

Comment:
same comment as the box shadow above to use `--pf-t--global--color--brand--default`